src/pylast/__init__.py

```diff
@@ -2244,38 +2244,7 @@ def get_name(self, properly_capitalized=False):
 
         return self.name
 
-    def get_artist_tracks(self, artist, cacheable=False, stream=False):
-        """
-        Deprecated by Last.fm.
-        Get a list of tracks by a given artist scrobbled by this user,
-        including scrobble time.
-        """
-
-        warnings.warn(
-            "User.get_artist_tracks is deprecated and will be removed in a future "
-            "version. User.get_track_scrobbles is a partial replacement. "
-            "See https://github.com/pylast/pylast/issues/298",
-            DeprecationWarning,
-            stacklevel=2,
-        )
-
-        params = self._get_params()
-        params["artist"] = artist
-
-        def _get_artist_tracks():
-            for track_node in _collect_nodes(
-                None,
-                self,
-                self.ws_prefix + ".getArtistTracks",
-                cacheable,
-                params,
-                stream=stream,
-            ):
-                yield self._extract_played_track(track_node=track_node)
-
-        return _get_artist_tracks() if stream else list(_get_artist_tracks())
-
-    def get_friends(self, limit=50, cacheable=False, stream=False):
+    def get_friends(self, limit=50, cacheable=False):
```

Comment:
This lost its `stream=False` in the merge

Reply:
Thx, fixing it now.